Scripts/copy-plugins.sh

```diff
@@ -16,20 +16,30 @@ function copy_plugins {
       echo Copying device plugin: $plugin to frameworks directory in app
       plugin_path="$(readlink "$f" || echo "$f")"
       plugin_as_framework_path="${BUILT_PRODUCTS_DIR}/${FRAMEWORKS_FOLDER_PATH}/${plugin%.*}.framework"
```

Comment:
The fix is the call to the function I added removeSimArchitectures. That will strip out the architectures for the plugin frameworks.

I also added some comments to help organize the code a bit as I understand it. believe the second half of this function deals with nested frameworks within the plugin.